{"name": "验证码输入错误，手机号、密码输入正确", "status": "passed", "attachments": [{"name": "log", "source": "9a1adeef-8c85-439a-a5ca-9eddbafb1122-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "title", "value": "'验证码输入错误，手机号、密码输入正确'"}, {"name": "inBody", "value": "{'password': '123456aa', 'phone': '18843211234', 'v_code': '123456'}"}, {"name": "expData", "value": "{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}"}], "start": 1647423166699, "stop": 1647423166872, "uuid": "f3148ca3-30b6-4563-9de9-aeb38f102d93", "historyId": "fecaedd06a989d6823ffc667120f1ff1", "testCaseId": "e9e06552f9fdd1f1f5eca42736142beb", "fullName": "testCases.test_RegisterCase.TestRegister#test_register", "labels": [{"name": "story", "value": "注册接口"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_RegisterCase"}, {"name": "subSuite", "value": "TestRegister"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "5836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_RegisterCase"}]}